docs/monitoring/autodiscovery.md

```diff
@@ -0,0 +1,60 @@
+title: Sematext Autodiscovery Monitoring
+description: Sematext Autodiscovery Monitoring automatically scans for services that can be monitored using Sematext Agent and, depending on your configuration, automatically sets up monitoring agents. 
+
+## Autodiscovery Monitoring
```

Comment:
This sounds like "Monitoring of Autodiscovery".
Options:
(Service) Autodiscovery and Monitoring
(Service) Autodiscovery for Monitoring